src/App.vue

```diff
@@ -412,6 +430,13 @@ export default {
     this.loadFromLocalStorage();
   },
   methods: {
+    updateDataPreview() {
+      this.previewDataStringify = JSON.stringify(this.previewData, null, 2);
+    },
```

Comment:
in order to remove the created on line 409 and to make this method to look nice and have the debounce functionality. I would write it like...
```suggestion
    updateDataPreview: debounce(function () {
      this.previewDataStringify = JSON.stringify(this.previewData, null, 2);
    }, 1000),
```

Reply:
if the following suggestion is approved. You can remove the created on line 409:411

Reply:
Fixed